cobrawap/__main__.py

```diff
@@ -303,56 +452,60 @@ def run(profile=None, extra_args=None, **kwargs):
     return None
 
 
-def run_stage(profile=None, stage=None, extra_args=None, **kwargs):
-    # select profile
+def run_stage(stage=None, profile=None, extra_args=None, **kwargs):
+    # # allow for positional stage argument
```

Comment:
```suggestion
    # allow for positional stage argument
```

Same as above: do we want to use positional arguments?

Reply:
see comment above

Reply:
See above.